### PR TITLE
chore: disable coverage in ci

### PR DESCRIPTION
**Pull request by bvanjoi**
close https://github.com/web-infra-dev/modern.js/issues/4780

### CODE REVIEW

**Comment by bvanjoi** [comment]
It can reduce the time cost.

before: https://github.com/web-infra-dev/modern.js/actions/runs/6557895579/job/17810369144?pr=4809

image

now: 

image

File: packages/builder/builder-cli/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "new": "modern new",
     "build": "modern-lib build",
     "dev": "modern-lib build --watch",
-    "test": "vitest run",
+    "test": "vitest run --no-coverage",
```
Comment (chenjiahan) [inline packages/builder/builder-cli/package.json:40]: Why not just remove https://github.com/web-infra-dev/modern.js/blob/main/scripts/vitest-config/src/index.ts#L8-L10

Comment (bvanjoi) [reply packages/builder/builder-cli/package.json:40]: I did not notice that.

Comment (chenjiahan) [reply packages/builder/builder-cli/package.json]: Haha it's a bit hidden

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**